### DISCUSSION TITLE
run server on Docker

### DISCUSSION THREAD

**Discussion by mohammadsadeghforoughi** [rcsoccersim/rcssserver#131] [Ideas]
Hi, 
I worked with this software 5 years ago and I only remember the pain of installing libs, building, and running on Linux distributions.
My idea is to add a Dockerfile to the project to run this project on any environment without any pain :))
I can add it to the source code by a PR.
What is your opinion about this?

**Comment by hidehisaakiyama** [comment]
I understand that Docker deployment can be beneficial. However, it is difficult to provide official support for Docker configurations because we do not currently have the resources to maintain them.
